Contribute/How-to-add-a-new-verion/index.md

```diff
@@ -0,0 +1,181 @@
+---
+versionFrom: 8.0.0
+---
+
+# How to add a new version of an article
+
+This tutorial will guide you through the steps it takes to add a new version of an existing article in the Umbraco Documentation.
+
+:::tip
+The **current major version** on the Umbraco Documentation is currently set to Umbraco 8 (v8). This means that any content that isn't targetting a specific version will be recognized as v8 content.
+
+If you want to help out with **updating our documentation against Umbraco 9** you can find more details about that process in the [Umbraco 9 Documentation](#umbraco-9-documentation) section at the bottom of this article.
+:::
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Forking and cloning the documentation repository](#forking-and-cloning-the-documentation-repository)
+* [Creating the new article](#creating-the-new-article)
+* [Adding labels to the article](#adding-labels-to-the-article)
+* [Updating the content](#updating-the-content)
+* [Creating the PR](#creating-the-pr)
+* [Umbraco 9 Documentation](#umbraco-9-documentation)
+* [Example: Creating a new version of an article targetting Umbraco 9](#example-creating-a-new-version-of-indexmd-targetting-an-upcoming-version-Umbraco-9)
+
+## Prerequisites
+
+To work with and contribute to the Umbraco Documentation there are a few tools and extensions we recommend that you use. Using these tools will ensure consistency throughout the documentation as well as help you create the best possible version of the article.
+
+* Visual Studio Code (VS Code)
+* Markdown linter - Extension for VS Code
+* Vale - Extension for VS Code
+
+We also highly recommend that you're familiar with Git and the Markdown syntax.
+
+## Forking and cloning the documentation repository
+
+The very first step is to get the documentation files down to your local machine, so you can start working with it. This is done by forking the UmbracoDocs repository and cloning it all down to your local machine.
+
+1. Follow our guide on [how to fork the UmbracoDocs repository](../Pull-Requests/#options-2-creating-a-pr-through-a-fork).
+2. Clone your fork to your local machine or open using GitHub Desktop.
+
+![Click on 'Code' to get the option to copy the clone URL.](images/clone-your-fork.png)
+
+Once you have a local clone of the UmbracoDocs repository on your local machine, open the entire `UmbracoDocs/` folder using your favorite editor.
+
+We recommend using Visual Studio Code as it can be extended with a lot of useful tools for writing documentation.
+
+## Creating the new article
```

Comment:
How about mentioning: If you want the article to display in a tree view, add the article inside a folder along with the images folder and the versioned .md files

Reply:
Ah! Good catch. Doesn't seem like I've mentioned anywhere, where to actually put the copy of the article 🤔 

I'll give it another look, and see if I can get it in there!

Reply:
I added a step to each of the two guides on creating the article.
Explains to put the new article in the same folder.